src/components/event-card/EventCard.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+    <div
+        class="card vs-event-card"
+        data-test="vs-event-card"
+    >
+        <div class="card-body">
+            <div
+                class="vs-event-card__header"
+                data-test="vs-event-card__header"
+            >
+                <VsHeading
+                    class="vs-event-card__header-heading"
```

Comment:
Just a heads up that with BEM, you don't need to nest the class names so it would just go:

vs-event-card__header > vs-event-card__heading or vs-event-card__date

Read more: https://en.bem.info/methodology/quick-start/#nesting-1